source/install/tarball.rst

```diff
@@ -4,29 +4,21 @@
 Installing Percona Server for MongoDB from Binary Tarball
 =========================================================
 
-You can find links to the binary tarballs from the
+You can find the link to the binary tarball under the *Generic Linux* menu item on the 
 `Percona Server for MongoDB download page <https://www.percona.com/downloads/percona-server-mongodb-3.6/>`_.
 
-1. Fetch and extract the correct binary tarball.
-   For example, if you are running Debian 8 ("jessie"):
-
-   .. code-block:: bash
-
-      wget https://www.percona.com/downloads/percona-server-mongodb-3.6/percona-server-mongodb-3.6.5-1.3/binary/debian/jessie/x86_64/percona-server-mongodb-3.6.5-1.3-ra1cb178-jessie-x86_64-bundle.tar
-      tar xfz percona-server-mongodb-3.6.5-1.3-ra1cb178-jessie-x86_64-bundle.tar
-
-2. Copy the extracted binaries to the target directory, for example:
-
+1. Fetch and extract the binary tarball.
+   
    .. code-block:: bash
 
-      mkdir ~/psmdb
-      cp -r -n percona-server-mongodb-3.6.5-1.3-ra1cb178-jessie-x86_64-bundle/bin ~/psmdb/
+      wget https://www.percona.com/downloads/percona-server-mongodb-3.6/percona-server-mongodb-3.6.18-6.0/binary/tarball/percona-server-mongodb-3.6.3-1-x86_64.glibc2.12.tar.gz
+      tar -xf percona-server-mongodb-3.6.3-1-x86_64.glibc2.12.tar.gz
```

Comment:
Filename should contain proper version name - percona-server-mongodb-3.6.19-7.0-x86_64.glibc2.12.tar.gz, the same version name should be in URL